.ci/pipeline/release_matrix_job.yaml

```diff
@@ -0,0 +1,62 @@
+---
+job: LIBXLIO-release
+# step_allow_single_selector: false
+registry_host: harbor.mellanox.com
+registry_auth: 1daaea28-800e-425f-a91f-3bd3e9136eea
+registry_path: /swx-infra/media/xlio
+
+kubernetes:
+  privileged: true
+  cloud: il-ipp-blossom-prod
+  nodeSelector: 'beta.kubernetes.io/os=linux'
+  namespace: swx-media
+  limits: '{memory: 8Gi, cpu: 7000m}'
+  requests: '{memory: 8Gi, cpu: 7000m}'
+
+env:
+  MAIL_FROM: jenkins@nvidia.com
+
+volumes:
+  # Default release location
+  - {mountPath: /auto/sw/release/sw_acceleration, hostPath: /auto/sw/release/sw_acceleration}
+  # User profile for release
+  - {mountPath: /var/home/swx-jenkins, hostPath: /labhome/swx-jenkins}
+
+runs_on_dockers:
+  - {file: '.ci/dockerfiles/Dockerfile.rhel8.6.release', name: 'rhel8.6', uri: '$arch/$name/release', build_args: '--no-cache', arch: 'x86_64', tag: '20240718'}
+
+
+steps:
+  - name: Build-dpcp
+    parallel: false
+    run: |
+      git clone https://github.com/Mellanox/libdpcp.git libdpcp
+      cd libdpcp
+      ./autogen.sh && ./configure --prefix=/usr && sudo make install
+
+  - name: Release
+    parallel: false
+    run: |
+      env WORKSPACE=$PWD .ci/do_release.sh
```

Comment:
`env WORKSPACE=$PWD` seems to be redundant
Jenkins already provides `WORKSPACE` variable which will be used as root of script execution (i.e. will be equal to `PWD`)